lib/phoenix/router/console_formatter.ex

```diff
@@ -120,13 +144,20 @@ defmodule Phoenix.Router.ConsoleFormatter do
         _ -> plug
       end
 
+    label =  
+      case Map.get(metadata, :label, "") do
+        label when label in [nil, ""] -> 
+          inspect(plug_opts)
+        label -> label
+      end
+
     String.pad_leading(route_name, route_name_len) <>
       "  " <>
       String.pad_trailing(verb, verb_len) <>
       "  " <>
       String.pad_trailing(path, path_len) <>
       "  " <>
-      "#{inspect(log_module)} #{inspect(plug_opts)}\n"
```

Comment:
This is printing the plug_opts, no?!

Reply:
Yes, but for all of our generated routes wouldn't the plug opts be *the same value*? Ohhhhhhhhhhhhh. You mean allow them to return *new* `plug_opts` to be displayed instead. So I'd return `plug_opts: [resource: Resource, action: Action]`. Sorry, I didn't understand.